app/api/index.mjs

```diff
@@ -2,14 +2,16 @@ import { dirname, join } from 'path'
 import url from 'url'
 import { readFileSync } from 'fs'
 
-
 /** @type {import('@enhance/types').EnhanceApiFn} */
 export async function get(req) {
   let here = dirname(url.fileURLToPath(import.meta.url))
   let base = join(here, 'posts.json')
   let posts = JSON.parse(readFileSync(base, 'utf-8'))
     .reverse()
 
+  let hCardPath = join(here, 'h-card.json')
```

Comment:
could we consistently use `let` or `const`? either one, just not both (unless its `const` and you need `let`)